packages/sveltekit/package.json

```diff
@@ -24,11 +24,13 @@
     "@sentry/node": "7.41.0",
     "@sentry/svelte": "7.41.0",
     "@sentry/types": "7.41.0",
-    "@sentry/utils": "7.41.0"
+    "@sentry/utils": "7.41.0",
+    "magic-string": "^0.30.0"
   },
   "devDependencies": {
-    "@sveltejs/kit": "^1.10.0",
-    "vite": "^4.0.0"
+    "@sveltejs/kit": "^1.5.0",
```

Comment:
^1.5.0 is identical to what what's set as a version in an auto-grenerated ootb SK project